doc/configcommands.dsv

```diff
@@ -8,7 +8,7 @@ bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This
 bookmark-autopilot||[yes/no]||no||If set to `yes`, the configured bookmark command is executed without any further input asked from user, unless the url or the title cannot be found/guessed.||bookmark-autopilot yes
 bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking plugin. See the documentation on bookmarking for further information.||bookmark-cmd "~/bin/delicious-bookmark.sh"
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
-browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` in <command> will be replaced by the URL being opened, enclosed in single quotes. Any occurrences of `%F` in <command> will be replaced by the feed's URL in single quotes.||browser "w3m %u"
+browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. For more information, see <<_handling_seperate_link_types>>.||browser "w3m %u"
```

Comment:
The man page doesn't include doc/newsboat.asciidoc, so Asciidoctor can't figure out the linked section's title and produces somewhat ugly output:

> … For more information, see [_handling_seperate_link_types]. …

Please add the title to the link itself. You can test this locally with `make doc && man doc/newsboat.1`.